backend/question_service/src/questionmanager.js

```diff
@@ -28,18 +28,19 @@ const connectToDB = () => {
   };
 
 const createQuestion = (request, response) => {
+  console.log(request.body);
   const {title, description, category, complexity} = request.body; //Extract attribs from request.
   const qn_num = parseInt(request.body.qn_num);
   if (!request.body || !qn_num || !title || !description || !category || !complexity) {
     const msg = {'msg': `All fields must be filled.`, 'qn_num': null};
     return response.status(400).json(msg);
   }
   
-  Question.create(request.body).then(() => {
+  Question.create({qn_num, title, description, category, complexity}).then(() => {
```

Comment:
Good catch on this. Request.body != {qn_num, title, description, category, complexity} all the time, esp if additional data are added.